### ISSUE TITLE
TinyMCE - elFinderBrowser not defined

### ISSUE THREAD

**Issue by Froxz** [Pseudoagentur/soa-sentinel#78]
```javascript
'file_picker_callback' => 'elFinderBrowser',
```
throws an js error
```javascript
ncaught ReferenceError: elFinderBrowser is not defined
```

On [tinymce-4x](https://github.com/barryvdh/laravel-elfinder#tinymce-4x)
Missing this function
```javascript
function elFinderBrowser (field_name, url, type, win) {
  tinymce.activeEditor.windowManager.open({
    file: '<?= route('elfinder.tinymce4') ?>',// use an absolute path!
    title: 'elFinder 2.0',
    width: 900,
    height: 450,
    resizable: 'yes'
  }, {
    setUrl: function (url) {
      win.document.getElementById(field_name).value = url;
    }
  });
  return false;
```

**Comment by bihlmaierh** [comment]
Hi @Froxz,

please use the following link to integrate TinyMCE (Part "TinyMCE 4.x") in your application:

> https://github.com/barryvdh/laravel-elfinder

When you use our TinyMCE in the current version please have attention to pull request https://github.com/barryvdh/laravel-elfinder/pull/85

I lock this issue now, because only the documentation needs to be updated.

Thanks for reporting!

**Comment by noxify** [comment]
@bihlmaierh - what's the status here?

**Comment by noxify** [comment]
@heikobihlmaier - please update the documentation for this!
I have removed it from the milestone and marked it as "known error".

**Comment by heikobihlmaier** [comment]
closed because #145
